modules/nf-core/multiqc/main.nf

```diff
@@ -0,0 +1,61 @@
+process MULTIQC {
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container 'docker://quay.io/microbiome-informatics/multiqc-with-dada2:latest'
```

Comment:
```suggestion
    container 'docker://quay.io/microbiome-informatics/multiqc-with-dada2:a1935b0'
```

I added a tag with the commit hash of the current latest commit on the plugin repo. Just so we have some kind of version pinning here to allow us to update in a managed way, rather than hoping `latest` remains backward compatible.